plugin/adsk/scripts/mayaUsdTranslatorImport.mel

```diff
@@ -54,18 +68,38 @@ global proc int mayaUsdTranslatorImport (string $parent,
 
         if (`exists usdImportDialog`)
         {
-            rowLayout -numberOfColumns 3 -cw 1 $cw1 -cat 1 "right" 0;
+            rowLayout -numberOfColumns 4 -cw 1 $cw1 -cat 1 "right" 0;
                 text -label "Scope and Variants: "
                     -al "right"
                     -sbm "Select a USD file and click Hierarchy View to open the Hierarchy View window. This window lets you toggle prim checkboxes and non-destructively switch between variants to build the scope of your import."
                     -ann "Select a USD file and click <strong>Hierarchy View</strong> to build the scope of your import and switch variants.<br><br><strong>Scope</strong>: A USD file consists of prims, the primary container object in USD. Prims can contain any scene element, like meshes, lights, cameras, etc. Use the checkboxes in the <strong>Hierarchy View</strong> to select and deselect prims to build the scope of your import.<br><br><strong>Variant</strong>: A variant is a single, named variation of a variant set. Each variant set is a package of alternatives that users can switch between non-destructively. A variant set has no limits to what it can store. Variants can be used to swap out a material or change the entire hierarchy of an asset. A single prim can have many variants and variant sets, but only one variant from each variant set can be selected for import into Maya."
                 ;
                 textField -ed false -nbg true -text "Please select a file to enable this option" -w $cw2 mayaUsdTranslator_SelectFileField;
-                button -label "Hierarchy View" -c "{string $usdFile = currentFileDialogSelection(); usdImportDialog $usdFile;}" mayaUsdTranslator_ViewHierBtn;
+                button -label "Hierarchy View" -c "mayaUsdTranslatorImport_ViewHierCB" mayaUsdTranslator_ViewHierBtn;
+                iconTextButton -w 32 -h 32 -command ("mayaUsdTranslatorImport_ClearImportData;") -
+                    style "iconAndTextVertical" -image1 "refresh.png" -label "" mayaUsdTranslator_ResetBtn;
+
+                string $nbPrimsInScopeLabel = "Total Number of Prims in Scope:";
+                string $nbModifiedVariantsLabel = "Variants Switched in Scope:";
 
                 // Hide the button by default.
                 button -e -visible false -w 1 mayaUsdTranslator_ViewHierBtn;
+                iconTextButton -e -visible false mayaUsdTranslator_ResetBtn;
+            setParent ..;
+
+            rowLayout -numberOfColumns 3 -cw 1 $cw1 -cat 1 "right" 0 -cal 1 "right" mayaUsd_NBPrimsInScopeRL;
+                text -label "";
+                text -label $nbPrimsInScopeLabel;
+                text -label "---" mayaUsd_NBPrimsInScopeLabel;
+            setParent ..;
```

Comment:
I added an empty text label at the start of these rowLayouts because it was just easier to line up with the UI above it.